python/lsst/daf/butler/cli/utils.py

```diff
@@ -19,9 +19,20 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import click
+import os
+
 from ..core.utils import iterable
 
 
+# DAF_BUTLER_MOCK is set by some tests as an environment variable and indicates
+# to the cli_handle_exception function that instead of executing the command
+# implementation function it should print details about the called command to
+# stdout. These details are then used to verify the command function was loaded
+# and received expected inputs.
+DAF_BUTLER_MOCK = {"DAF_BUTLER_MOCK": ""}
```

Comment:
It reads like the variable here is the contents of the environment variable but environment variables can't be a dict so is the key in the dict the environment variable name and the value in the dict what is set by the user? Having the global use the same name as the key is confusing here.